multipass/how-to/how-to-set-up-a-graphical-interface.md

```diff
@@ -15,30 +15,43 @@ The graphical desktop can be viewed in various ways. In this document, we descri
 
 The images used by Multipass do not come with a graphical desktop installed. For this reason, a desktop environment must be installed (here we use `ubuntu-desktop` but there are as many other options as flavors of Ubuntu exist), along with the RDP server (we will use `xrdp` but there are also other options such as `freerdp`). 
 
-To do this, first you need to log in to the running Multipass instance:
+To do this, first you need to choose a Multipass instance. List your instances:
 
 ```plain
-$ multipass shell headbanging-squid
+multipass list
+```
+
+Sample output:
+
+```plain
+Name                    State             IPv4             Image
+headbanging-squid       Running           10.49.93.209     Ubuntu 22.04 LTS
+```
+
+Next log in to the running Multipass instance:
+
+```plain
+multipass shell headbanging-squid
 ```
 
 Once inside the instance, run the following commands to install `ubuntu-desktop` and `xrdp`:
 
 ```plain
-$ sudo apt update
-$ sudo apt install ubuntu-desktop xrdp
+sudo apt update
+sudo apt install ubuntu-desktop xrdp
 ```
 
 Then, we need a user with a password in order to log in. One possibility is setting a password for the default `ubuntu` user:
 
 ```plain
-$ sudo passwd ubuntu
+sudo passwd ubuntu
 ```
 
 You will be asked to enter and re-enter a password. 
 
 You are done on the server side!
 
-Now, quit the Ubuntu shell on the instance with the `logout` command and find out in the host the IP address to connect to:
+Now, quit the Ubuntu shell on the instance with the `logout` command and find out in the host the IP address to connect to. The IP address is in the output of `multipass list` from the first step:
 
 ```plain
 $ multipass list
```

Comment:
```suggestion
multipass info headbanging-squid
```

Reply:
If we choose to suggest using this command, the sample output below should also be updated accordingly.

Reply:
I will run the list and add the output.